articles/data-factory/quickstart-create-data-factory-azure-cli.md

```diff
@@ -121,20 +121,19 @@ Next, create a linked service and two datasets.
 
    ```json
    {
-       "type": 
-           "AzureBlob",
-           "linkedServiceName": {
-               "type":"LinkedServiceReference",
-               "referenceName":"AzureStorageLinkedService"
-               },
-           "annotations": [],
-           "type": "Binary",
-           "typeProperties": {
-               "location": {
-                   "type": "AzureBlobStorageLocation",
-                   "fileName": "emp.txt",
-                   "folderPath": "input",
-                   "container": "adftutorial"
+       "type": "AzureBlob",
```

Comment:
Cross checked with https://docs.microsoft.com/en-us/azure/data-factory/quickstart-create-data-factory-rest-api#create-datasets. It seems `"type": AzureBlob",` is redundant.